packages/cli/commons/src/analytics/identifier.ts

```diff
@@ -43,19 +43,13 @@ export class Identifier {
         user_id: userId,
         device_id: deviceId,
         ...this.getAmplitudeBaseEventProperties(),
-        ...this.getOrganizationIdentifier(),
```

Comment:
Removing organization info from user identification and moving it to event properties